### ISSUE TITLE
Request to deviate from iOS Mail app behavior (touch two buttons at once)

### ISSUE THREAD

**Issue by yepher** [CEWendel/SWTableViewCell#216]
## TLDR;

When a cell is swiped open SWTableViewCell should disallow two buttons to be pressed at once.
## Background

I have found that responding to two touches on different controls at once is generally problematic. (Hint to testers) This is a really good way to find crashes in a lot of apps while testing. 

There is another issue #148 for SWTableViewCell that is similar but when two cells are touched. I too had to manage that case.

Now, I am struggling with the case where two buttons are pressed on SWTableViewCell at once.
## Example of problem

In iOS Mail if you swipe a message cell open and press "Archive" and with a second finger "More" buttons at nearly the same time you will get an archived message and an action menu for that message. This is almost certainly not what the user had intended. In this case it is probably best to just throw out both touches but at the very least only one should have gotten handled.

I anticipate that a lot of apps that use SWTableViewCell have a similar problem or worse just crash.

I consider this a bug and expect iOS Mail will be updated at some point to resolve this. 

I propose this project leads the way and does it first in the internals of SWTableViewCell.

 Thoughts?
